src/components/Header/Header.jsx

```diff
@@ -1,17 +1,27 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import styles from './Header.module.css';  
+import React, { useState } from "react";
+// import { Link } from 'react-router-dom';
+import icon_close from "../../assets/icon-close.svg";
+import logo from "../../assets/logo.svg";
+import styles from "./Header.module.css";
 
 const Header = () => {
+  const [mobileMenu, setMobileMenu] = useState(false);
+  const toggleMenu = () => {
```

Comment:
```suggestion
 const toggleMenu = () => {
    setMobileMenu((toggleMenu) => !toggleMenu);
  };
```

Reply:
We just need to reverse the value so this can be of less code

Reply:
See https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/pull/24/commits/49b796c9c657122a39dbfcac09176ec80a692af9#r1685844601